vivisect/defconfig.py

```diff
@@ -110,15 +133,38 @@
             'pointertables':{
                 'table_min_len':'How many pointers must be in a row to make a table?',
             },
+            'stack':{
+                'base':'Stack base address',
+                'mask':'Stack mask',
+                'top':'Stack top address',
+                'pointer':'Stack pointer',
+            },
+            'taint':{
+                'base':'Taint base address',
+                'byte':'Taint byte value',
+                'offset':'Taint VA offset',
+                'mask':'Taint mask',
+            },
         },
+
+        'arch':{
+            'ppc':{
+                'options':'PowerPC processor features to enable',
+                'bootstrap':{
+                    'rchwaddrs':'A list of addresses to look for at for the reset-control half word (RCHW) used for PowerPC boot target identification',
+                },
+                'findvlepages':'Flag to search and automatically add VLE memory map pages from MMU instructions',
+                'mmu':'A list of [<address>, <size>] values that indicate memory segments where PowerPC VLE instructions can be found',
```

Comment:
?

Reply:
I've changed `mmu` to `vle_pages` in `vivisect/defconfig.py`